.github/vale/Docker/Substitute.yml

```diff
@@ -46,7 +46,6 @@ swap:
   repo: repository
   scroll: navigate
   (?:sign on|log on|log in|logon|login): sign in
-  sign up: register
```

Comment:
should we add the opposite rule in?
`register: sign up`?

Reply:
Was on the fence as it seems like register in the docs is also used for things other than create account (e.g. `Register a domain` or `Register a service`) so I wasn't sure if the recommendation would be additional noise or not 🤔 Happy to update this though!

Reply:
Ah, good point, ignore me!